src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -162,10 +163,10 @@ function BaseOnboardingAccounting({shouldUseNativeStyles, route}: BaseOnboarding
                         onboardingCompanySize,
                         userReportedIntegration,
                     );
-
-                    Welcome.setOnboardingAdminsChatReportID();
-                    Welcome.setOnboardingPolicyID();
-
+                    InteractionManager.runAfterInteractions(() => {
```

Comment:
Please add a note explaining why we're using InteractionManager.runAfterInteractions here.